package.json

```diff
@@ -58,6 +57,8 @@
   "browserslist": {
     "production": [
       ">0.2%",
+      "not ie <= 99",
```

Comment:
didn't find the specific source of these lines (found on another issue), but this is the problem it's fixing: https://web3auth.io/community/t/big-int-error-in-react-production-build/3289